### PR TITLE
Fixed Server.cpp

### PR DESCRIPTION
**Pull request by Creaprog**
Performance improvement.

### CODE REVIEW

**Comment by NiLSPACE** [comment]
Is there a reason why LOG isn't used here?

**Comment by madmaxoft** [comment]
ProtoProxy had no logging at the time this piece of code was written. I guess we can now use it, since the logging framework has been pulled in already.

**Comment by SphinxC0re** [comment]
Wait..! We have a logging framework? Where is it documented?

**Comment by Creaprog** [comment]
Puts is simpler than printf but be aware that the former automatically appends a newline.

**Comment by madmaxoft** [comment]
@SphinxC0re the LOG family of functions that are normally used by the server, they write to the console and into a logfile. Yes, we've had that since before I joined the project, which means long ago :)

@Creaprog change the calls to `LOG`, for style unification.

**Comment by Creaprog** [comment]
I replaces puts by LOG ?

**Comment by SphinxC0re** [comment]
Yes :dancer:

**Comment by Creaprog** [comment]
Log is auto newline ?

**Comment by NiLSPACE** [comment]
Yeah

**Comment by NiLSPACE** [comment]
Anyone any idea why the build failed?

**Comment by SphinxC0re** [comment]
It says
```error: ‘LOG’ was not declared in this scope```

**Comment by Creaprog** [comment]
Yes replaces puts by LOG is error. Why use function LOG ?
The function log is no write, where is the function LOG ?

**Comment by worktycho** [comment]
LOG should be declared in Logger.h. The main point of LOG is that it can be forwarded to places other than stdout. At the moment in outputs to a file as well but there is the potential to output LOG to plugins or hooks for android rather than reading the log file if we get that working again.

**Comment by Creaprog** [comment]
Ok, i add Logger.h.

**Comment by worktycho** [comment]
Logger.h should already have been included through globals.h. Also you're breaking protoproxy which defines the logging functions differently. You're problem is that the protoproxy globals only define LOGINFO, LOGWARNING and LOGERROR. You'll need to add a define to tools/protoproxy/Globals.h

File: Tools/ProtoProxy/Server.cpp [hunk 1]
```diff
@@ -85,7 +85,7 @@ int cServer::Init(short a_ListenPort, short a_ConnectPort)
 
 void cServer::Run(void)
 {
-	printf("Server running.\n");
+	LOG("Server running.");
```
Comment (bearbin) [inline Tools/ProtoProxy/Server.cpp]: You missed one log here, so the build failed.

**Comment by Creaprog** [comment]
@bearbin Sorry, i have forget 1 LOG.

**Comment by bearbin** [comment]
Build worked, merging.

**Comment by Creaprog** [comment]
Yes, thanks :)

**Pull request merged**